vanilla-app/package.json

```diff
@@ -14,15 +14,20 @@
     "vite": "4.3.9"
   },
   "dependencies": {
-    "@justeat/pie-design-tokens": "5.7.0",
-    "@justeattakeaway/pie-button": "0.34.0",
-    "@justeattakeaway/pie-card-container": "0.11.0",
-    "@justeattakeaway/pie-cookie-banner": "0.7.0",
-    "@justeattakeaway/pie-css": "0.6.0",
-    "@justeattakeaway/pie-divider": "0.8.0",
-    "@justeattakeaway/pie-icon-button": "0.18.0",
-    "@justeattakeaway/pie-icons-webc": "0.6.0",
-    "@justeattakeaway/pie-modal": "0.28.0",
-    "@justeattakeaway/pie-toggle-switch": "0.14.0"
+    "@justeat/pie-design-tokens": "5.8.2",
+    "@justeattakeaway/pie-button": "latest",
+    "@justeattakeaway/pie-card": "latest",
+    "@justeattakeaway/pie-cookie-banner": "latest",
+    "@justeattakeaway/pie-css": "latest",
+    "@justeattakeaway/pie-divider": "latest",
+    "@justeattakeaway/pie-icon-button": "latest",
+    "@justeattakeaway/pie-icons-webc": "latest",
+    "@justeattakeaway/pie-modal": "latest",
+    "@justeattakeaway/pie-switch": "latest",
+    "lit": "2.7.5"
+  },
+  "volta": {
+    "node": "18.18.2",
+    "yarn": "1.22.19"
```

Comment:
RE what @siggerzz mentioned, feel free to bump this to 3.5.0 if it's no overhead :)

Reply:
It wasn't just a one-line change so I'll leave it for now